src/app/Library/CrudPanel/Traits/Relationships.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel\Traits;
+
+use Exception;
+use Illuminate\Database\Eloquent\Relations\Relation;
+
+trait Relationships
+{
+    /**
+     * Check if the given method exists in current crud model.
+     *
+     * @param string $methodName
+     * @return ReflectionMethod|bool
+     */
+    public function modelHasMethod($methodName)
+    {
+        return method_exists($this->model, $methodName);
+    }
+
+    /**
+     * Provided a field name we try to figure if the name is a relation name or it's a database table field that points to any relation.
+     *
+     * @param string $fieldName
+     * @return array|bool
+     */
+    public function getRelationFromFieldName($fieldName)
+    {
+        if ($this->modelHasMethod($fieldName)) {
```

Comment:
I see this is the only place where we use ```modelHasMethod()```. Correct me if I'm wrong.

These two do the same thing:
```php
$this->modelHasMethod($fieldName);
method_exists($this->model, $fieldName);
```

And I think the second is just as easy to read & understand (if not more), than the first. So I'm not sure a separate method for this is necessary. I propose we eliminate ```modelHasMethod()``` in favour of using the PHP standard equivalent.

Reply:
Sure, atm it is the only place where we use it. Even if we use it multiple times using method_exists would work just the same and fine. 

Nice spot.

Reply:
Done! Pushing soon.